### ISSUE TITLE
PlotFillAboveBelow, with custom baseline same color even if below the baseline. 

### ISSUE THREAD

**Issue by el-aasi** [swharden/ScottPlot#477]
So, what I assumed is that by providing a custom baseline, fill that would be below the custom baseline will be of another color (i.e. Red) but instead it is the same color as the above one (Green), even if the graph acts based on the custom baseline. I assume that it is somewhat tied to negative values and/or 0. Is there a chance to adjust that somehow?

**Comment by swharden** [comment]
Hey @el-aasi, thank you for reporting this issue! It looks like the `baseline` argument is not being properly handled.

```cs
plt.PlotFillAboveBelow(xs, ys, fillAlpha: .5, labelAbove: "above", labelBelow: "below", baseline: 3.21);
```

![image](https://user-images.githubusercontent.com/4165489/85733182-fb5bca80-b6c9-11ea-8557-48c52e418b3e.png)

**Comment by swharden** [comment]
Hey @el-aasi, I fixed this with 6d585f5 and the change will be in the next (version 4.0.38) release. Thanks again for reporting this!

![image](https://user-images.githubusercontent.com/4165489/85734059-b5ebcd00-b6ca-11ea-9c19-5dc592e236cf.png)

**Comment by el-aasi** [comment]
@swharden thanks for the quick reply.
